userspace/libscap/engine/modern_bpf/CMakeLists.txt

```diff
@@ -16,6 +18,8 @@ if(NOT MODERN_BPF_SKEL_DIR)
   add_subdirectory(${LIBSCAP_DIR}/driver/modern_bpf ${CMAKE_BINARY_DIR}/driver/modern_bpf)
 else()
```

Comment:
[nit] maybe also add an informational message showing the full or relative path the skeleton header file (not just dir) was written to?

Reply:
done, ty :)